src/components/Game/Wordsy/Utils/helpers.js

```diff
@@ -1,5 +1,7 @@
 export const getOrthographyPattern = (orthography) => {
-  const SORTED_ORTHOGRAPHY = orthography?.sort((a, b) => b.length - a.length)
+  const SORTED_ORTHOGRAPHY = orthography
+    ?.slice()
+    .sort((a, b) => b.length - a.length)
```

Comment:
The original `orthography` object is what the keyboard reads for ordering the buttons so we need to use slice to make a copy, ensuring the original is unmodified. 

The sorted orthography is needed to ensure the longer characters are tested for a match first in the game logic. This is important when longer custom characters contain other sub-characters. eg: "bc" and "b" are both characters in the custom alphabet.